pkg/cloud/cloud.go

```diff
@@ -66,7 +66,7 @@ const (
 	// MinTotalIOPS represents the minimum Input Output per second.
 	MinTotalIOPS = 100
 	// MaxTotalIOPS represents the maximum Input Output per second.
-	MaxTotalIOPS = 20000
+	MaxTotalIOPS = 64000
```

Comment:
Would it be more future proof to let the sdk return an error if the passed in parameters are invalid? That avoids needing to update the driver every time the aws limits change.

Reply:
Yes, I think you're right. I'll push another commit.